src/components/time/ReadableTime.stories.js

```diff
@@ -3,48 +3,38 @@ import * as React from 'react';
 import ReadableTime from './ReadableTime';
 import notes from './ReadableTime.stories.md';
 
-export const relativeTimestamps = () => {
-    const msInADay = 24 * 60 * 60 * 1000;
-    const oneHourInMs = 60 * 60 * 1000;
+const msInADay = 24 * 60 * 60 * 1000;
+const oneHourInMs = 60 * 60 * 1000;
```

Comment:
small but since this is top level now, we could use uppercase

```
const MILLISECONDS_PER_HOUR = 60 * 60 * 1000;
const MILLISECONDS_PER_DAY = MILLISECONDS_PER_HOUR * 24;
const MILLISECONDS_PER_WEEK = MILLISECONDS_PER_DAY * 7;
```